cmake/Modules/FindoneMKL.cmake

```diff
@@ -0,0 +1,24 @@
+set(ONEMKL_LIBRARIES)
+
+if(DEFINED ENV{MKLROOT})
+  SET(INTEL_MKL_DIR $ENV{MKLROOT})
+else()
+  SET(INTEL_MKL_DIR "/opt/intel/oneapi/mkl/latest")
```

Comment:
Do we have a variable for it after sourcing setvars.sh in bundle?
Like for SYCL we have, CMPLR_ROOT
 <img width="292" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/0b53ed85-cd7a-4798-bad9-895a8566bf03">
Hard-code here is not compatible, if the bundle changes directories in future.

Reply:
One question: where can we get the ENV `SYCL_ROOT`? AFAK, BaseKit and Bundle provide `CMPLR_ROOT`.

Reply:
It's an option to user. `SYCL_ROOT` is PyTorch defined. `CMPLR_ROOT` is tool kit defined.

Reply:
Assuming oneMKL is always installed together with the compiler in bundle, `MKL_ROOT` will be obtained relatively based on `CMPLR_ROOT`.